Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -373,7 +373,7 @@ void BluetoothReal::WaitForHCICommandComplete(const u16 opcode)
   for (int tries = 0; tries < 100; ++tries)
   {
     if (libusb_interrupt_transfer(m_handle, HCI_EVENT, buffer.data(),
-                                  static_cast<int>(buffer.size()), &actual_length, 20) == 0 &&
+                                  static_cast<int>(buffer.size()), &actual_length, TIMEOUT) == 0 &&
```

Comment:
I changed the number of tries to 10 and kept the timeout to TIMEOUT.
I believe it is better to non-officially allow some adapters to work over the network than to impose arbitrary limits that could prevent them from working where they could have worked otherwise.